core/authorship/impl/block_builder_impl.hpp

```diff
@@ -32,7 +30,7 @@ namespace kagome::authorship {
    private:
     primitives::BlockHeader block_header_;
     std::shared_ptr<runtime::BlockBuilderApi> r_block_builder_;
-    common::Logger logger_;
+    common::Logger logger_ = common::createLogger("BlockBuilder");
```

Comment:
probably we should move logger initialization to constructor impl. like it was done in other files.

Reply:
for consistency.